$OpenBSD: patch-examples_cgi-bin_dump-env_sh,v 1.2 2015/09/18 14:37:52 kspillner Exp $

Drop Bash and tweak to work with our ksh.  Merged upstream in PR#166.
--- examples/cgi-bin/dump-env.sh.orig	Thu Jul 16 11:52:56 2015
+++ examples/cgi-bin/dump-env.sh	Thu Jul 16 11:58:52 2015
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 # Copyright 2013 Joe Walnes and the websocketd team.
 # All rights reserved.
@@ -36,7 +36,8 @@ echo
 
 for NAME in ${NAMES}
 do
-	echo ${NAME}=${!NAME:-<unset>}
+	eval "value=\${${NAME}}"
+	env -i "${NAME}=${value:-<unset>}"
 done
 
 # Additional HTTP headers
